core/src/main/java/io/grpc/internal/MutableHandlerRegistry.java

```diff
@@ -29,7 +29,9 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.grpc;
+package io.grpc.internal;
```

Comment:
Closing the loop on this ... discussed offline and decided that the registries should get moved back to the top-level, but be marked experimental.